django/db/models/expressions.py

```diff
@@ -1025,7 +1025,13 @@ def __init__(self, queryset, output_field=None, **extra):
 
     def __getstate__(self):
         state = super().__getstate__()
-        state.pop('_constructor_args', None)
+        args, kwargs = state['_constructor_args']
+        if args:
+            queryset = args[0]
+            # Prevent the QuerySet from being evaluated.
+            if queryset is not None:
```

Comment:
Is there cases where `queryset` is not provided? Maybe because it was provided using `kwargs`?

Reply:
No, it will always be in `args`.

Reply:
Then why is the `if queryset is not None` branch necessary?

Reply:
Right, it's not necessary. Thanks, removed.

Reply:
@charettes OK it's possible :disappointed: , I will add an extra test for it.